src/bp/admin/ui/src/management/licensing/index.tsx

```diff
@@ -274,11 +284,12 @@ class LicenseStatus extends React.Component {
   }
 
   render() {
-    return this.props.loading ? <LoadingSection /> : this.renderBody()
```

Comment:
Why remove the loading component?

Reply:
loading wasn't implemented so it was never actually used